internal/stack/_static/fleet-server-healthcheck.sh

```diff
@@ -2,16 +2,19 @@
 
 set -e -o pipefail
 
+NUMBER_SUCCESSES=$1
+WAITING_TIME=$2
+
 healthcheck() {
-    curl --cacert /etc/ssl/elastic-agent/ca-cert.pem -f https://localhost:8220/api/status | grep -i healthy 2>&1 >/dev/null
+    curl -s --cacert /etc/ssl/elastic-agent/ca-cert.pem -f https://localhost:8220/api/status | grep -i healthy 2>&1 >/dev/null
 }
 
 # Fleet Server can restart after announcing to be healthy, agents connecting during this restart will
 # fail to enroll. Expect 3 healthy healthchecks before considering it healthy.
-expected=3
-for i in $(seq $expected); do
+for i in $(seq "$NUMBER_SUCCESSES"); do
+	echo "Iter $i"
 	healthcheck
-	sleep 1
+	sleep "$WAITING_TIME"
 done
```

Comment:
Create a separate PR for these changes related to Fleet Server healthcheck https://github.com/elastic/elastic-package/pull/1867